style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,200&display=swap');
+
+.home {
+  display: grid;
+  grid-template-rows: 92px 810px;
+  width: 375px;
+}
+
+.header {
+  background-color: #3c3a39;
+}
+
+.header h1 {
+  color: #fff;
+  position: relative;
+  margin-top: 56px;
+  margin-bottom: 12px;
+  margin-left: 24px;
+  font-size: 18px;
+}
+
+.header a {
+  position: absolute;
+  color: #fff;
+  width: 24px;
+  height: 24px;
+  top: 66px;
+  left: 362px;
+}
+
+.headline {
```

Comment:
As the project requirements specifically ask for, we need to layout the elements in the headline with the use of Flexbox;

Screenshot from the project requirements:
![image](https://user-images.githubusercontent.com/83596135/164488475-664434ac-6674-4881-9fae-865404a6e02c.png)

- [ ] Kindly, make use of Flexbox in order to display and style the elements of the headline